packages/design-system/src/scalars/components/phid-field/phid-field.tsx

```diff
@@ -126,7 +124,7 @@ const PHIDFieldRaw = React.forwardRef<HTMLInputElement, PHIDFieldProps>(
             {label}
           </FormLabel>
         )}
-        {autoComplete ? (
+        {autoComplete && fetchOptionsCallback && fetchSelectedOptionCallback ? (
```

Comment:
the `fetchSelectedOptionCallback` should not determine if the input can have autocomplete or not, it should be used to refresh the selected item data only. With `autoComplete && fetchOptionsCallback` condition should be enough here